composer.json

```diff
@@ -19,7 +19,6 @@
         "php-64bit": ">=7.1.0"
```

Comment:
Could probably bump the version, even, right? I mean, I have no skin in this game anymore haha, but I think 7.1 is pretttty old. 😛